src/Form/CoverType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Trick;
+use App\Entity\TrickMedia;
+use App\Repository\TrickMediaRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CoverType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
```

Comment:
I don't understand what you try to do

Reply:
CoverType is a form used to select wich image will be used as a primary image (cover) for a trick, it have un radio button and the corresponding image as label, it's totally different as the 'cover' field in the TrickType where it is a FileType used to upload an image as a cover when creating a new trick.
Below I'm using an EvenListener to display only the images attached to the current trick